app/views/content/is-teaching-right-for-me/physics/_article.html.erb

```diff
@@ -0,0 +1,120 @@
+<div class="row inset">
+<section class="col col-720">
+  <p>
+    Teaching physics is an exciting and rewarding career. Schools need more specialist physics teachers, so it's a great choice of subject.
+  </p>
+
+  <p>
+You could inspire your pupils to consider rewarding careers from tackling climate change to artificial intelligence (AI).
+  </p>
+  <p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free bursaries of £28,000 and scholarships of £30,000 are available for eligible trainee physics teachers</a>.</p>
```

Comment:
@caitlin-reidy same as computing. I think we should highlight bursaries are available but I don't think we should link to the bursaries info here. We've got the link to it in the funding section which makes more sense. And we shouldn't have it on the page twice. I don't think it has to go in the 1st mention of something. GDS says links should be used in context https://www.gov.uk/guidance/content-design/links

Reply:
i replied to your comment about this yesterday, i cant help disagree with this. I guess if we change it to a generic line about bursaries and scholarships available it'd make sense not to link it but then i think thats a bad user journey anyway, making them hunt for more info on it. I think if anything it should be the first mention that links out. But if you still want it changed i can do